doc/api-and-naming.md

```diff
@@ -41,17 +41,17 @@ deprecated in.
 the shell's environment.
 
-|                                     | `bash_completion`       | `completions/*`                                                                       |
-|:------------------------------------|:------------------------|:--------------------------------------------------------------------------------------|
-| public configuration variables      | `BASH_COMPLETION_*`     | `BASH_COMPLETION_CMD_${Command^^}_${Config^^}`                                        |
-| private non-local variables         | `_comp__*`              | `_comp_cmd_${Command}__${Data}`                                                       |
-| private non-local mutable variables | `_comp__*_mut_*`        | `_comp_cmd_${Command}__mut_${Data}`                                                   |
-| exporter function local variables   | `_*` (not `_comp*`)     | `_*` (not `_comp*`)                                                                   |
-| public/exported functions           | `_comp_*`               | `_comp_cmd_${Command}` (functions for `complete -F`)                                  |
-|                                     |                         | `_comp_xfunc_${Command}_${Utility}` (functions for use with `_comp_xfunc`)            |
-|                                     | `_comp_compgen_${Name}` | `_comp_xfunc_${Command}_compgen_${Name}` (generators for use with `_comp_compgen -x`) |
-| private/internal functions          | `_comp__*`              | `_comp_cmd_${Command}__${Utility}` (utility functions)                                |
-|                                     |                         | `_comp_cmd_${Command}__compgen_${Name}` (generators for use with `_comp_compgen -i`)  |
+|                                     | `bash_completion`   | `completions/*`                                                                       |
+|:------------------------------------|:--------------------|:--------------------------------------------------------------------------------------|
+| public configuration variables      | `BASH_COMPLETION_*` | `BASH_COMPLETION_CMD_${Command^^}_${Config^^}`                                        |
+| private non-local variables         | `_comp__*`          | `_comp_cmd_${Command}__${Data}`                                                       |
+| private non-local mutable variables | `_comp__*_mut_*`    | `_comp_cmd_${Command}__mut_${Data}`                                                   |
+| exporter function local variables   | `_*` (not `_comp*`) | `_*` (not `_comp*`)                                                                   |
+| public/exported functions           | `_comp_*`           | `_comp_xfunc_${Command}_${Utility}` (functions for use with `_comp_xfunc`)            |
+| - completors (for `complete -F`)    | `_comp_complete_*`  | `_comp_cmd_${Command}`                                                                |
```

Comment:
After this, `_comp_complete_load` is in an incorrect namespace as it does something else, we should look into renaming it to something else.

Reply:
In my thinking, `_comp_complete_load` is still in the same class of "completion functions" in the sense that `_comp_complete_load` is intended to be specified to `complete -F`.

Actually, when I introduced the name `_comp_complete_load` in [`d9082d2` (\#1032)](https://github.com/scop/bash-completion/pull/1032/commits/d9082d2c8dff6b709786862bcd1b8d1698648ea1), I already anticipated the changes of this PR and made consistent with this PR. This is also one of the reasons that I split `_completion_loader` into two functions with different purposes, `_comp_load` (for a utility to purely load a completion) and `_comp_complete_load` (for `complete -F`) (as in comment https://github.com/scop/bash-completion/pull/1032/commits/d9082d2c8dff6b709786862bcd1b8d1698648ea1#r1295226734).